src/Sentry/Protocol/Dsn.cs

```diff
@@ -12,138 +10,56 @@ namespace Sentry
     /// <remarks>
     /// <see href="https://docs.sentry.io/quickstart/#configure-the-dsn"/>
     /// </remarks>
-    public class Dsn
+    public sealed class Dsn
     {
-        private readonly string _dsn;
+        public string OriginalString { get; }
 
-        /// <summary>
-        /// The project ID which the authenticated user is bound to.
-        /// </summary>
         public string ProjectId { get; }
 
-        /// <summary>
-        /// An optional path of which Sentry is hosted.
-        /// </summary>
         public string? Path { get; }
 
-        /// <summary>
-        /// The optional secret key to authenticate the SDK.
-        /// </summary>
         public string? SecretKey { get; }
```

Comment:
Why get rid of all the docs?

Reply:
I'll bring them back before "undrafting" the PR.